code/modules/unit_tests/serving_tray.dm

```diff
@@ -5,7 +5,7 @@
 	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
 	var/obj/structure/table/the_table = allocate(/obj/structure/table)
 	var/obj/item/storage/bag/tray/test_tray = allocate(/obj/item/storage/bag/tray)
-	var/obj/item/reagent_containers/food/banana = allocate(/obj/item/reagent_containers/food/snacks/grown/banana)
+	var/obj/item/reagent_containers/food/banana = allocate(/obj/item/food/grown/banana)
```

Comment:
Change this to another food that is an `/obj/item/reagent_containers/food/snacks`--the point of this test is to ensure both old food and new food work on serving trays, and we already have a new food underneath (`/obj/item/food/the_bread`).